spec.html

```diff
@@ -37301,7 +37419,7 @@ <h1>JSON.parse ( _text_ [ , _reviver_ ] )</h1>
       <p>The *"length"* property of the `parse` function is *2*<sub>𝔽</sub>.</p>
       <emu-note>
         <p>Valid JSON text is a subset of the ECMAScript |PrimaryExpression| syntax. Step <emu-xref href="#step-json-parse-validate"></emu-xref> verifies that _jsonString_ conforms to that subset, and step <emu-xref href="#step-json-parse-assert-type"></emu-xref> asserts that that parsing and evaluation returns a value of an appropriate type.</p>
-        <p>However, because <emu-xref href="#sec-__proto__-property-names-in-object-initializers"></emu-xref> applies when evaluating ECMAScript source text and does not apply during `JSON.parse`, the same source text can produce different results when evaluated as a |PrimaryExpression| rather than as JSON. Furthermore, the Early Error for duplicate *"__proto__"* properties in object literals, which likewise does not apply during `JSON.parse`, means that not all texts accepted by `JSON.parse` are valid as a |PrimaryExpression|, despite matching the grammar.</p>
+        <p>However, because <emu-xref href="#sec-object-initializer-static-semantics-early-errors"></emu-xref> applies when evaluating ECMAScript source text and does not apply during `JSON.parse`, the same source text can produce different results when evaluated as a |PrimaryExpression| rather than as JSON. Furthermore, the Early Error for duplicate *"__proto__"* properties in object literals, which likewise does not apply during `JSON.parse`, means that not all texts accepted by `JSON.parse` are valid as a |PrimaryExpression|, despite matching the grammar.</p>
```

Comment:
This is now wrong in 2 ways:
(1) The link is to an Early Errors clause, but Early Errors don't apply when *evaluating* ECMAScript source; the relevant link is to the runtime semantics.
(2) Even if you fix that, the sentence would be saying that the whole of PropertyDefinitionEvaluation doesn't apply during `JSON.parse`, which is not true. Instead, the clause applies both inside and outside `JSON.parse`, it just operates slightly differently.
```suggestion
        <p>However, because <emu-xref href="#sec-runtime-semantics-propertydefinitionevaluation"></emu-xref> behaves differently during `JSON.parse`, the same source text can produce different results when evaluated as a |PrimaryExpression| rather than as JSON. Furthermore, the Early Error for duplicate *"__proto__"* properties in object literals, which likewise does not apply during `JSON.parse`, means that not all texts accepted by `JSON.parse` are valid as a |PrimaryExpression|, despite matching the grammar.</p>
```

Pre-existing nits while I'm here (I'm not requesting for these to be fixed, just noting them for later):
(a) The phrase "evaluated as a |PrimaryExpression| rather than as JSON" is a false dichotomy. In both cases, it's evaluated as a |PrimaryExpression|. The dichotomy is whether or not the evaluation is happening at the behest of `JSON.parse`.
(b) It seems like it would be more logical to have the Early Errors sentence before the evaluation sentence.